src/optimise1.jl

```diff
@@ -0,0 +1,52 @@
+using IntervalArithmetic, DataStructures
+
+import Base.isless
+
+struct IntervalMinima{T<:Real}
```

Comment:
"Minimum" is the singular;  "minima" is the plural.

Reply:
You want "minimum" everywhere.

Reply:
Okay. Using `global_minimum` instead since `minimum` is an existent binding.